README.md

```diff
@@ -1,2 +1,183 @@
-# Hello-Microverse
-This repo is created for Day 2 activity of Microverse Full-Time web developer course.
+<a name="readme-top"></a>
+
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+
+
+<div align="center">
+  <br/>
+  <h3><b>Hello Microverse HTML Project README</b></h3>
+
+</div>
+
```

Comment:
- I am not sure this is supposed to be a part of the readme. Can we make it a bit more professional? Why don't we take it off entirely or replace it with a personal logo? What do you think? 🤗

Reply:
@Favourezeugwa you are totally right. Thanks for your time. I will take it off and create a new PR.

Reply:
Actually I couldn't see that when I check my README file. How could it be? So I can't resolve it. Do you have any idea or old experience about this.

Reply:
I deleted my README file from the main branch and cleared my Project description. Let's see.

Reply:
> I deleted my README file from the main branch and cleared my Project description. I hope this will solve the problem.
> 
> Should I start a new pull request?

![image](https://user-images.githubusercontent.com/63915024/211897164-e6f93ad4-f2ec-4324-9430-ed61a6eab25d.png)